api/src/services/opportunities_v1/get_opportunity.py

```diff
@@ -35,8 +40,46 @@ def _fetch_opportunity(
     return opportunity
 
 
+def get_boto_session() -> boto3.Session:
+    is_local = bool(os.getenv("IS_LOCAL_AWS", False))
+    if is_local:
+        return boto3.Session(aws_access_key_id="NO_CREDS", aws_secret_access_key="NO_CREDS")
+
+    return boto3.Session()
+
+
+def pre_sign_opportunity_file_location(
+    opp_atts: List,
+) -> List[OpportunityAttachment]:
+    s3_config = S3Config()
+
+    s3_client = get_s3_client(s3_config, get_boto_session())
+    for opp_att in opp_atts:
+        file_loc = opp_att.file_location
+        bucket, key = split_s3_url(file_loc)
+        pre_sign_file_loc = s3_client.generate_presigned_url(
+            "get_object",
+            Params={"Bucket": bucket, "Key": key},
+            ExpiresIn=s3_config.presigned_s3_duration,
+        )
+        if s3_config.s3_endpoint_url:
+            pre_sign_file_loc = pre_sign_file_loc.replace(
+                s3_config.s3_endpoint_url, "http://localhost:4566"
+            )
```

Comment:
I'd suggest we add a comment calling out this hacky fix as only mattering locally due to docker path issues.

Reply:
Should i mark it as todo as well ?